sda-commons-server-key-mgmt/src/test/java/org/sdase/commons/keymgmt/KeyMgmtBundleWithDefaultTest.java

```diff
@@ -0,0 +1,134 @@
+package org.sdase.commons.keymgmt;
+
+import static io.dropwizard.testing.ResourceHelpers.resourceFilePath;
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import io.dropwizard.core.Application;
+import io.dropwizard.core.setup.Bootstrap;
+import io.dropwizard.core.setup.Environment;
+import io.dropwizard.testing.ConfigOverride;
+import io.dropwizard.testing.junit5.DropwizardAppExtension;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Order;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.RegisterExtension;
+import org.sdase.commons.keymgmt.manager.KeyMapper;
+import org.sdase.commons.keymgmt.manager.MapOrFailKeyMapper;
+
+class KeyMgmtBundleWithPlaceholderTest {
+
+  @RegisterExtension
+  @Order(0)
+  static final DropwizardAppExtension<KeyMgmtBundleTestConfig> DW =
+      new DropwizardAppExtension<>(
+          KeyMgmtBundleTestApp.class,
+          resourceFilePath("test-config.yml"),
+          ConfigOverride.config("keyMgmt.apiKeysDefinitionPath", resourceFilePath("keys")),
+          ConfigOverride.config("keyMgmt.mappingDefinitionPath", resourceFilePath("mappings")));
+
+  private static KeyMgmtBundle<KeyMgmtBundleTestConfig> keyMgmtBundle;
+
+  @BeforeAll
+  static void init() {
+    KeyMgmtBundleTestApp app = DW.getApplication();
+    keyMgmtBundle = app.getKeyMgmtBundle();
+    assertThat(keyMgmtBundle).isNotNull();
+  }
+
+  @Test
+  void shouldThrowIfMappingDoesNotExists() {
+    assertThrows(
+        IllegalArgumentException.class, () -> keyMgmtBundle.createKeyMapper("NOTEXISTING"));
+  }
+
+  @Test
+  void shouldGetKeyMappingCaseInsensitive() {
```

Comment:
It's probably not really necessary to repeat everything that is tested in the other test class, no?

Reply:
Removed most old tests and added a few new ones.